e2e/packages/image/src/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "@frontity/components/image";
+import Package from "../types";
+
+type ImageProps = Parameters<typeof Image>["0"];
+
+const props: ImageProps = {
+  alt: "gullfoss",
+  src: "https://test.frontity.io/wp-content/uploads/2016/11/Gullfoss.jpg",
```

Comment:
Could you please use a local images? I don't want this test to fail if test.frontity.io is offline or if those images are removed. You can download them and add them to the `"image"` package.